src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -82,11 +89,18 @@
  *
  */
 public class OpenShiftOAuth2SecurityRealm extends SecurityRealm {
+	private static final Logger LOGGER = Logger.getLogger(OpenShiftOAuth2SecurityRealm.class.getName());
 
     /**
      * OAuth 2 scope. This is enough to call a variety of userinfo api's.
      */
     private static final String SCOPE = "user:info";
+    
+    private static final String DEFAULT_SVC_ACCT_DIR = "/run/secrets/kubernetes.io/serviceaccount";
+    private static final String SA_DIR = "serviceAccountDirectory";
+    private static final String C_ID = "clientId";
```

Comment:
`C` isn't something I auto-complete to client.  Can I get this one spelled? (the other abbreviations work in my head)